assets/src/edit-story/migration/migrations/v0002_dataPixelTo1080.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const NEW_PAGE_WIDTH = 1080;
+const NEW_PAGE_HEIGHT = 1920;
+const OLD_PAGE_WIDTH = 412;
+const OLD_PAGE_HEIGHT = 732;
+
+const SCALE_X = NEW_PAGE_WIDTH / OLD_PAGE_WIDTH;
+const SCALE_Y = NEW_PAGE_HEIGHT / OLD_PAGE_HEIGHT;
+
+function dataPixelTo1080( { pages, ...rest } ) {
+	return {
+		pages: pages.map( reducePage ),
+		...rest,
+	};
+}
+
+function reducePage( { elements, ...rest } ) {
+	return {
+		elements: elements.map( updateElement ),
+		...rest,
+	};
+}
+
+function updateElement( { x, y, width, height, fontSize, ...rest } ) {
+	const element = {
+		x: dataPixels( x * SCALE_X ),
+		y: dataPixels( y * SCALE_Y ),
+		width: dataPixels( width * SCALE_X ),
+		height: dataPixels( height * SCALE_Y ),
+		...rest,
+	};
+	if ( typeof fontSize === 'number' ) {
+		element.fontSize = dataPixels( fontSize * SCALE_Y );
+	}
+	return element;
+}
+
+/**
+ * See `units/dimensions.js`.
```

Comment:
Good idea to copy this function over 👍

Reply:
It'd be good to explicitly prohibit use of import dependencies outside this package here. Would that be possible?

Reply:
Good idea!

Try the following ESLint rule:

```json
{
    "import/no-restricted-paths": [ "error", {
      "zones": [ {
        "target": "./assets/src/edit-story/migration/migrations",
        "from": "./assets/src/edit-story",
        "except": [ "./migration/migrations" ]
      } ]
    } ]
}
```

Reply:
The `except` part makes it possible to import the migration functions in the tests, but the migration functions cannot import anything else (e.g. importing `dataPixels` would raise an error)

Reply:
Awesome! Works!